### PR TITLE
implemented  shows in dashboard

### PR DESCRIPTION
**Pull request by Kate-github**
- added seed data
- renamed show to TvShow
- Added to dashboard.

### CODE REVIEW

File: controllers/dashboard-routes.js [hunk 1]
```diff
@@ -36,14 +39,29 @@ router.get('/', withAuth, (req, res) => {
...
             }, summaryData);
-            res.render('dashboard', {
-                services,
-                loggedIn: req.session.loggedIn,
-                summary: summaryData
+            TvShow.findAll({
+                where: {
+                    // use the ID from the session after testing (req.session.user_id)
+                    premiereDate: {
+                        [Op.gte]: moment()
```
Comment (hellovolcano) [inline controllers/dashboard-routes.js:51]: why is this changed to Op when the rest of these are declared sequelize in the branch?

File: models/index.js [hunk 2]
```diff
@@ -39,4 +39,4 @@ User_Subscription.belongsTo(User, {
     foreignKey: 'user_id'
 })
 
-module.exports = { Service, Show, User, User_Subscription };
+module.exports = { Service, TvShow: TvShow, User, User_Subscription };
```
Comment (hellovolcano) [inline models/index.js:42]: I think this can just be TvShow not TvShow: TvShow ?

File: models/TvShow.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
 const {Model,DataTypes}=require('sequelize');
 const sequelize=require('../config/connection');
 
-class Show extends Model{}
+class TvShow extends Model{}
 
-Show.init(
+TvShow.init(
```
Comment (hellovolcano) [inline models/TvShow.js:6]: I think we ought to associate the user with the tv show that is added and then use that to filter the shows for the dashboard, or else it's going to display every tv show under the sun in the user's personal dashboard

**Review by hellovolcano — COMMENTED** [review]
Awesome that it works! Probably good enough for the demo with some specific stuff, but we still need to add a way for the user to add/edit/remove their shows, and a way to associate the show with a user. (Simplest way for this project would be a one user to many shows relationship vs. a many-to-many)

Also needs to be styled on the frontend; not sure if you were planning on doing that or not.

File: models/index.js (hunk 2 above)
Comment (Kate-github) [reply models/index.js:42]: TvShow is just short hand, is this going to block the PR?

File: models/TvShow.js (hunk 3 above)
Comment (Kate-github) [reply models/TvShow.js:6]: well the requirement says that it's colored if the subscription is active. we can filter by active subscriptions but then we wouldn't need to color code it.

Comment (Kate-github) [reply models/TvShow.js:6]: image

File: controllers/dashboard-routes.js (hunk 1 above)
Comment (Kate-github) [reply controllers/dashboard-routes.js:51]: this is what the documentation shows how to do "greater than".

**Comment by Kate-github** [comment]
> Awesome that it works! Probably good enough for the demo with some specific stuff, but we still need to add a way for the user to add/edit/remove their shows, and a way to associate the show with a user. (Simplest way for this project would be a one user to many shows relationship vs. a many-to-many)
> 
> Also needs to be styled on the frontend; not sure if you were planning on doing that or not.

This wasn't discussed and not in the requirements. It's not going to happen by tomorrow.

File: models/TvShow.js (hunk 3 above)
Comment (hellovolcano) [reply models/TvShow.js:6]: Not saying there's anything wrong with your current implementation; it all looks correct -- but the user dashboard should display things specific to the user.

**Review by hellovolcano — APPROVED** [review]
